Makefile

```diff
@@ -138,7 +138,7 @@ bin: $(GOCMDS) $(GOINSTR)
 regen:       ## Refresh autogenerated files and reformat code.
              ## Use CHECK=1 to only validate clean repo after run.
              ##
-regen: $(ADDLICENSE)
+regen: $(ADDLICENSE) $(HELM)
```

Comment:
Helm isn't used in this target? At least not directly...and this won't install it in the Docker container.

Reply:
It is used, but you are right, it's done via importing env bingo source, thanks! Will kill.